Gems/MiniAudio/Code/Source/Clients/MiniAudioListenerComponent.cpp

```diff
@@ -46,6 +46,17 @@ namespace MiniAudio
                 ->Attribute(AZ::Script::Attributes::Category, "MiniAudio Listener")
                 ->Event("SetPosition", &MiniAudioListenerRequests::SetPosition)
                 ->Event("SetFollowEntity", &MiniAudioListenerRequests::SetFollowEntity)
+                ->Event("Get Channel Count", &MiniAudioListenerRequests::GetChannelCount)
+                ->Event("Get Inner Cone Angle In Radians", &MiniAudioListenerRequests::GetInnerAngleInRadians)
+                ->Event("Set Inner Cone Angle In Radians", &MiniAudioListenerRequests::SetInnerAngleInRadians)
+                ->Event("Get Inner Cone Angle In Degrees", &MiniAudioListenerRequests::GetInnerAngleInDegrees)
+                ->Event("Set Inner Cone Angle In Degrees", &MiniAudioListenerRequests::SetInnerAngleInDegrees)
+                ->Event("Get Outer Cone Angle In Radians", &MiniAudioListenerRequests::GetOuterAngleInRadians)
+                ->Event("Set Outer Cone Angle In Radians", &MiniAudioListenerRequests::SetOuterAngleInRadians)
+                ->Event("Get Outer Cone Angle In Degrees", &MiniAudioListenerRequests::GetOuterAngleInDegrees)
+                ->Event("Set Outer Cone Angle In Degrees", &MiniAudioListenerRequests::SetOuterAngleInDegrees)
+                ->Event("Get Percent Volume Outside Outer Cone", &MiniAudioListenerRequests::GetOuterVolume)
+                ->Event("Set Percent Volume Outside Outer Cone", &MiniAudioListenerRequests::SetOuterVolume)
```

Comment:
Event names with spaces are unintuitive to reference in LUA, that's why all the others match the function names.
```suggestion
                ->Event("GetChannelCount", &MiniAudioListenerRequests::GetChannelCount)
                ->Event("GetInnerAngleInRadians", &MiniAudioListenerRequests::GetInnerAngleInRadians)
                ->Event("SetInnerAngleInRadians", &MiniAudioListenerRequests::SetInnerAngleInRadians)
                ->Event("GetInnerAngleInDegrees", &MiniAudioListenerRequests::GetInnerAngleInDegrees)
                ->Event("SetInnerAngleInDegrees", &MiniAudioListenerRequests::SetInnerAngleInDegrees)
                ->Event("GetOuterAngleInRadians", &MiniAudioListenerRequests::GetOuterAngleInRadians)
                ->Event("SetOuterAngleInRadians", &MiniAudioListenerRequests::SetOuterAngleInRadians)
                ->Event("GetOuterAngleInDegrees", &MiniAudioListenerRequests::GetOuterAngleInDegrees)
                ->Event("SetOuterAngleInDegrees", &MiniAudioListenerRequests::SetOuterAngleInDegrees)
                ->Event("GetOuterVolume", &MiniAudioListenerRequests::GetOuterVolume)
                ->Event("SetOuterVolume", &MiniAudioListenerRequests::SetOuterVolume)
```

The correct way to set the even name and parameters for Script Canvas is to use the Script Canvas names generation tools,  I wasn't able to find any documentation on this but there should be a menu in the Script Canvas tool for generating node name meta data files, but you may need to enable the Script Canvas developer gem - pinging @lsemp3d 

Alternately, you can provide that meta data in the C++
https://docs.o3de.org/docs/user-guide/scripting/script-canvas/programmer-guide/behavior-context/#best-practice-displaying-ebus-event-parameter-names-in-nodes

Reply:
Please have a look at b2d8349953494850c9f73aa129f7930994de6e94, hopefully that's right.

Reply:
@AMZN-alexpete @lsemp3d It looks like b2d8349953494850c9f73aa129f7930994de6e94 didn't do what was intended. There are no spaces in the names of the nodes in SC and the tooltips aren't showing up. Also, when comparing them to other nodes I noticed that they will display which category they belong to underneath their name/title, how is that achieved? See here:
![image](https://github.com/o3de/o3de/assets/3263382/88c895ed-7bda-4afb-a1d7-6ba893db777b)

Reply:
I found the documentation for generating the translation data for node meta data:  https://docs.o3de.org/docs/user-guide/scripting/script-canvas/editor-reference/text-replacement/#example-generating-a-translation-file-to-replace-text

Reply:
Thanks @AMZN-alexpete, I got it working with a2dc3ca1383a685a0d429ea13ecf6a666ec21325.